### PR TITLE
feat:Colorpallete

### PR DESCRIPTION
**Pull request by amithire1201**
### Description
New Gradient Pallete is added for background
https://trello.com/c/jQP4nQ8W/51-gradient-color-pallete-usage
### Screenshots

image

### Types of changes

New feature (non-breaking change which adds functionality) 

### How has this been tested?
Add Advanced column Block/Section/Child Column and then select background type as gradient under background setting in right pane.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/GradientSettings/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		gradientColor2,
+		gradientLocation1,
+		gradientLocation2,
+		gradientPosition,
+		gradientType
+	} = attributes;
+
+    
+	//set the gradient value to settings if new gradientvalue is not set
+	if (gradientValue === "") {
```
Comment (RajkiranBagal) [inline src/components/GradientSettings/index.js:19]: Use Yoda Conditions.

**Comment by RajkiranBagal** [comment]
@amithire1201 Perform PHPCS

**Comment by RajkiranBagal** [comment]
@amithire1201 Add changelog

File: src/blocks/columns/attributes.js [hunk 2]
```diff
@@ -300,6 +300,11 @@ const attributes = {
 		type: "string",
 		default: "outset"
 	},
+	gradientValue: {
+		type: "string",
+		default:"",
```
Comment (RajkiranBagal) [inline src/blocks/columns/attributes.js:305]: If you do not want any default value, you can remove the param itself

Also, Add attribute in class-uagb-config.php also.

File: src/components/GradientSettings/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import hexToRgba from "../../../dist/blocks/uagb-controls/hexToRgba";
```
Comment (RajkiranBagal) [inline src/components/GradientSettings/index.js:1]: Change the Folder name from "GradientSettings" to "gradient-settings"

File: src/blocks/columns/attributes.js (hunk 2 above)
Comment (amithire1201) [reply src/blocks/columns/attributes.js:305]: I need this attribute with empty default.

File: src/components/GradientSettings/index.js (hunk 1 above)
Comment (amithire1201) [reply src/components/GradientSettings/index.js]: Done

File: src/components/GradientSettings/index.js (hunk 3 above)
Comment (amithire1201) [reply src/components/GradientSettings/index.js]: Done

**Pull request merged**